config/settings.yml

```diff
@@ -1725,9 +1721,8 @@ avs:
   api_jwt: abcd1234abcd1234abcd1234abcd1234abcd1234
 
 brd:
-  api_key: 'fake_key'
```

Comment:
@reiting Change these back to single quotes as it's an unnecessary update for this PR.